frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -61,6 +61,12 @@
     "CNI_PLUGIN_LABELS": "{{brokers.cni_plugin_labels}}",
     {{/service.virtual_networks}}
 
+    {{#service.tls}}
+    "ENABLE_TLS": "yes",
+    "BROKER_PORT_TLS": "{{brokers.port_tls}}",
+    "TASKCFG_ALL_KAFKA_ENABLE_TLS": "{{service.tls}}",
```

Comment:
If you wanted, ENABLE_TLS could be removed in favor of having svc.yml's templating use `TASKCFG_ALL_KAFKA_ENABLE_TLS` directly. Then it'd be a single value across the board.